spec/invoice_item_spec.rb

```diff
@@ -0,0 +1,145 @@
+require 'rspec'
+require './lib/invoice_item'
+
+describe InvoiceItem do
+  describe '#initialize' do
+    it 'exists' do
+      ii_details = {
+        id: 6,
+        item_id: 7,
+        invoice_id: 8,
+        quantity: 1,
+        unit_price: BigDecimal(10.99, 4),
```

Comment:
For ref -- you're doing this here in the details, but in the Item class, I had to add it to my #initialize method to get this working. Like the init method was: `@unit_price = BigDecimal(details[:unit_price])`